### PR TITLE
make landing page form section responsive, adjustments to hamburger menu per Danila

### PR DESCRIPTION
**Pull request by rileydanejohnston**
Main additions:
- media queries for form section
- adjustments to hamburger menu so it doesn't push the whole page down as it opens
- fix the pointer part of the tool tips

The main thing I don't know how to reconcile is the tool tips. Basically when you activate the tool tip on certain buttons they will be cut off at the edge of the screen. There are 2 reasons for this: 
   1.) the design has them pretty wide and they aren't included in the responsive part of the Figma file.
   2.) Overflow: hidden is on the App component to help with the purple background images at smaller screen sizes. Removing overflow: hidden creates horizontal scroll.

I'm not sure how to fix this without changing the design. If you have any recommendations I would love to hear them. 

- [ ] My code is covered by unit tests 
- [ x] This PR is less than [300 lines of code](https://www.atlassian.com/blog/git/written-unwritten-guide-pull-requests)
- [x ] My code [smells good](https://en.wikipedia.org/wiki/Code_smell) 
- [ x] I have tested my PR locally and proven it works. 
- [ x] There is [no copy-paste](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself)

### CODE REVIEW

**Comment by mentalcaries** [comment]
Hamburger menu is smooth!

I know it's different on the design, but it seems like it would make sense if the tool tips were facing the other direction, i.e towards the middle of the screen vs the edge of it

File: frontend/src/components/landing-help-icon/styledHelpIcon.js [hunk 1]
```diff
@@ -35,18 +40,21 @@ export const HelpWrapper = styled.div`
...
+  height: 28px;
+  top: -22px;
+  left: 20px;
+  z-index: 4;
+  opacity: ${(props) => (props.$isOpen === false ? 0 : 1)};
+  visibility: ${(props) => (props.$isOpen === false ? 'hidden' : 'visible')};
+`;
+
 export const HelpText = styled(SmallText)`
   color: #FFF;
```
Comment (mentalcaries) [inline frontend/src/components/landing-help-icon/styledHelpIcon.js:59]: Text here might need to be aligned to the left. Right now it's centred

File: frontend/src/components/landing-form/styledLandingForm.js [hunk 2]
```diff
@@ -81,17 +130,46 @@ export const Button = styled.button`
     background: #580abc;
     transition: background 0.3s;
   }
+
+  @media (max-width: 1023px) {
```
Comment (McCambley) [inline frontend/src/components/landing-form/styledLandingForm.js:134]: This is really weird, the button border radius seems like it doesn't appear as rounded as it does in the figma, yet all the styling and code is correct. the figma buttons look more like if you were to set this border-radius to 28px... but I'm not sure what's causing the difference in appearance.

File: frontend/src/components/landing-form/styledLandingForm.js [hunk 3]
```diff
@@ -66,8 +116,7 @@ export const Button = styled.button`
   height: 60px;
   border: none;
   border-radius: 20px;
-  margin: 20px 20px 0 0;
-  padding: 15px 30px;
+  padding: 20px 30px;
   background: #7000ff;
   box-shadow: 4px -4px 4px 0px #38008066 inset;
```
Comment (McCambley) [inline frontend/src/components/landing-form/styledLandingForm.js:121]: The second box shadow negates the first, combine them into one line.

File: frontend/src/components/header/styledHeader.js [hunk 4]
```diff
@@ -21,6 +21,12 @@ export const Wrapper = styled.header`
     flex-direction: column;
     align-items: stretch;
     padding: 12px 10px;
+    width: 96.25%;
+    position: absolute;
```
Comment (McCambley) [inline frontend/src/components/header/styledHeader.js:25]: Nice! this looks really good.

**Review by McCambley — APPROVED** [review]
Hey Riley,
Looks good. I left a comment for a few fixes, but I'll approve this and those can be included in a new PR that can deal with tooltips. As for the tooltip process. I don't have the answer, but I do have a lead. 

1) [This SO](https://stackoverflow.com/questions/50970336/prevent-css-tooltip-from-going-out-of-page-window/51067379) post talks about using `getBoundingClientRect()` to see if an object sits outside of the page. If it does, you can't translate it the amount needed to bring it back into pane. 
2) given that the tooltip will move irrespective of the mouse, and you might want to keep the tail on the mouse, you can use `evt.pageX` and `evt.pageY` (I think) and set the position of the tail via those on `mouseOver`... maybe? [Here's an SO post](https://stackoverflow.com/questions/3234256/find-mouse-position-relative-to-element) about that stuff.

Given that that might be a process, and I know you have some stuff branching off this branch, I'll approve this and you can do what you need to do to keep your working space the way that makes the most sense.

File: frontend/src/components/landing-form/styledLandingForm.js (hunk 3 above)
Comment (rileydanejohnston) [reply frontend/src/components/landing-form/styledLandingForm.js:121]: well how about that? Nice

File: frontend/src/components/landing-form/styledLandingForm.js (hunk 2 above)
Comment (rileydanejohnston) [reply frontend/src/components/landing-form/styledLandingForm.js:134]: Yeah I'm not sure either. I did have some perspective issues with the buttons when I started the markup. On the figma file they looked normal but when I coded them they looked freakishly large.

**Pull request merged**